pkg/i18n/english.go

```diff
@@ -1860,5 +1863,13 @@ func EnglishTranslationSet() TranslationSet {
 			AppendingLineToFile:      "Appending '{{.line}}' to file '{{.filename}}'",
 			EditRebaseFromBaseCommit: "Beginning interactive rebase from '{{.baseCommit}}' onto '{{.targetBranchName}}",
 		},
+		BreakingChangesTitle: "Breaking Changes",
+		BreakingChangesMessage: `You are updating to a new version of lazygit which contains breaking changes. Please review the notes below and update your configuration if necessary.
+For more information, see the full release notes at <https://github.com/jesseduffield/lazygit/releases>.`,
+		BreakingChangesByVersion: map[string]string{
+			"0.40.0": "- Some older message, just for testing", // to be removed
+			"0.41.0": `- The 'c' key is no longer the default key for copying commits to cherry pick. Bla bla bla
+- Squashing fixups using shift-S brings up a menu, and squashes all fixups in current branch if you hit enter`,
+		},
```

Comment:
@jesseduffield These are very preliminary. I don't have time right now to collect which messages we want to show, but in particular you will be much better at actually phrasing them, so I hope you don't mind if I leave this to you. I set the PR to draft for that reason.